### PR TITLE
[TOOLS-687] updateIssueBuffer: Parsed Retrofit Exception Message

### PR DESCRIPTION
**Pull request by jhonypg**

### CODE REVIEW

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [IssueBufferService.java#L103](https://github.com/objective-solutions/taskboard/blob/9b7eb6d0b7e461ee026fd1fe2ded5edac6ba379d/core/src/main/java/objective/taskboard/issueBuffer/IssueBufferService.java#L103): Refactor this method to reduce its Cognitive Complexity from 18 to the 15 allowed. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AS3776)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

File: core/src/main/java/objective/taskboard/issueBuffer/IssueBufferService.java [hunk 1]
```diff
@@ -115,8 +115,11 @@ public void updateIssueBuffer() {
                     try {
                         jiraIssueService.searchProject(projectKey, newProjectVisitor);
                         cardsRepo.addProject(projectKey);
-                    } catch (Exception ex) {
-                        log.warn("Exception when updating issue buffer", ex);
+                    } catch (RetrofitError e) {
```
Comment (herbert-venancio) [inline core/src/main/java/objective/taskboard/issueBuffer/IssueBufferService.java]: change `e` to `ex`

Comment (jhonypg) [reply core/src/main/java/objective/taskboard/issueBuffer/IssueBufferService.java]: Doine

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [IssueBufferService.java#L103](https://github.com/objective-solutions/taskboard/blob/f0c7d10cc4af5709c926c294fdfa128883a35da0/core/src/main/java/objective/taskboard/issueBuffer/IssueBufferService.java#L103): Refactor this method to reduce its Cognitive Complexity from 18 to the 15 allowed. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AS3776)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Pull request merged**